### PR TITLE
[stable10] Backport fixes of German translations

### PR DESCRIPTION
**Pull request by PVince81**
There were some mixed up between "förmlich" and "persönlich".
This was fixed by me on Transifex, synced down and this PR is the backport for stable10.

Fixes https://github.com/owncloud/core/issues/33454

### CODE REVIEW

File: settings/l10n/de.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ OC.L10N.register(
     "Invalid mail address" : "Ungültige E-Mail-Adresse",
     "A user with that name already exists." : "Ein Benutzer mit diesem Namen existiert bereits.",
     "Unable to create user." : "Benutzer konnte nicht erstellt werden.",
-    "Your %s account was created" : "Dein %s-Konto wurde erstellt",
+    "Your %s account was created" : "Ihr %s-Konto wurde erstellt",
```
Comment (PVince81) [inline settings/l10n/de.js]: arghhh no...

Comment (PVince81) [reply settings/l10n/de.js]: I've now changed it in this commit and also fixed on Transifex:

**Review by IljaN — APPROVED** [review]
Not 100% sure if "Deinen" should be capitalized. But probably yes..

**Comment by PVince81** [comment]
@IljaN I saw it capitalized in other translations and went for that, making it consistent

**Pull request merged**